crates/holochain_sqlite/src/sql.rs

```diff
@@ -1,4 +1,23 @@
 pub mod sql_cell {
+    // SELECT queries to select hashes of ops pending integration.
```

Comment:
All these additional select queries are copied from the update queries below and converted to a select.